Update routing_content_core_6.partial.md (#4124)
Particular/docs.particular.net: transports/msmq/routing_content_msmqtransport_[1,).partial.md

Mapping the logical destination to the physical address containing the queue and host names is the responsibility of physical routing. 

> 

Reviewer: Notpick but if the note is the same in all 3 places we should pull it into an "include"

https://github.com/Particular/docs.particular.net#markdown-includes

## Instance mapping file

NOTE: When using instance mapping: 
* The settings will have no effect on **audit and error queues**.

> * **publish/subscribe**: The publisher will be publishing messages only to the endpoint instances that have subscribed to the events, ignoring the settings in the mapping file (the adress of the subscriber that was provided in the subscription messgae will be used).

Reviewer: **publish/subscribe** => **publishing**

> * **publish/subscribe**: The publisher's instances should be entered in the Instance mapping file (so the subscribers can send a subscription message to them)

Reviewer: **publish/subscribe** => subscriptions
```suggestion
* **subscribing**: Subscription messages will be sent to all publisher instances listed in the Instance mapping file
```
